src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/ToolStripMenuItemTests.cs

```diff
@@ -138,10 +139,50 @@ public void ToolStripMenuItem_SetShortcutKeys(Keys keys)
     [InlineData(Keys.Control | Keys.Alt | Keys.Shift)]
     public void ToolStripMenuItem_SetShortcutKeys_ThrowsInvalidEnumArgumentException(Keys keys)
     {
-        using var item = new SubToolStripMenuItem();
+        using SubToolStripMenuItem item = new();
         Assert.Throws<InvalidEnumArgumentException>(() => item.ShortcutKeys = keys);
     }
 
+    [WinFormsTheory]
+    [MemberData(nameof(CultureInfo_Shortcut_TestData))]
+    public void ToolStripMenuItem_SetShortcutKeys_ReturnExpectedShortcutText(CultureInfo cultureInfo, CultureInfo cultureUIInfo, string shortCutText)
+    {
+        CultureInfo _uiCulture = Thread.CurrentThread.CurrentUICulture;
+        CultureInfo _curCulture = Thread.CurrentThread.CurrentCulture;
```

Comment:
typically `_` prefix is for fields
```suggestion
        CultureInfo uiCulture = Thread.CurrentThread.CurrentUICulture;
        CultureInfo curCulture = Thread.CurrentThread.CurrentCulture;
```